src/containers/PageContainer.jsx

```diff
@@ -63,7 +63,7 @@ class PageContainer extends React.PureComponent {
     const options = widget ? widget.options : null;
     const Tag = this.layouts[layout || 'default'];
     const WidgetTag = widget ? this.widgetTags[widget.type] : null;
-    const MediaTag = WidgetTag || image;
+    const MediaTag = WidgetTag; /*  || image */
```

Comment:
Instead of setting MediaTag = WidgetTag, let's go all in on WidgetTag.  That is to say, let's pass the layout components a prop called WidgetTag instead of MediaTag and get rid of this line altogether.

Reply:
Change made. As I was making these updates, I came to the same conclusion, but left it just in case there was some special case reason that you had created it like this.